### DISCUSSION TITLE
possible to disable volume control entirely?

### DISCUSSION THREAD

**Discussion by iogitio** [MonitorControl/MonitorControl#1108] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I have an LG Ultrawide and use SoundSource to control the monitor volume.

Is it possible to not have MonitorControl touch anything related to volume? Don't want to get it mixed up with SoundSource.

Doesn't look like it's possible to disable.

**Comment by waydabber** [comment]
It is possible. Simply hide the volume slider in the app and disable keyboard volume control. You can also disable volume DDC control on a per display basis under Preferences/Displays.

Hope this helps!
